src/PIDController.cpp

```diff
@@ -0,0 +1,12 @@
+#include "PIDController.h"
+
+PIDController::PIDController(double p, double i, double d) {
+    p_, i_, d_ = p, i, d;
```

Comment:
probably should use better names for these
- p, i, d would suggest these are the terms, but the values here are just the constant multipliers
- k_p, k_i, k_d would probably be more descriptive